capstone/src/main/java/com/google/sps/servlets/PollServlet.java

```diff
@@ -16,25 +16,32 @@
 import com.google.appengine.api.datastore.Query;
 import com.google.gson.Gson;
 import com.google.sps.Objects.Option;
+import com.google.sps.Objects.response.PollResponse;
 
 @WebServlet("/poll")
 public class PollServlet extends HttpServlet {
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String userId = "200";
     Query query = new Query("Option");
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     PreparedQuery results = datastore.prepare(query);
     List<Option> options = new ArrayList<Option>();
+    List<Long> votedOptions = new ArrayList<Long>();
     for (Entity entity : results.asIterable()) {
       long id = entity.getKey().getId();
       String text = (String) entity.getProperty("text");
       List<String> votes = (ArrayList<String>) entity.getProperty("votes");
       Option option = new Option(id, text, votes);
       options.add(option);
+      // If current user voted for option, add to list of voted options
+      if (votes != null && votes.contains(userId)) {
+        votedOptions.add(id);
+      }
```

Comment:
I think I'm confused but why is voted options a list? Can't a user only vote for one option at a time?

Reply:
This is to keep track of the options that the current user has voted for so that when the page is loaded, I use that information to mark certain checkboxes as checked

Reply:
I see, so to clarify, the current user is able to check multiple options?

Reply:
Yep